README.md

```diff
@@ -21,6 +21,7 @@ import logging
 
 logsense_token = getenv('LOGSENSE_TOKEN', None)
 logging.getLogger().addHandler(LogSenseHandler(logsense_token))
+logging.getLogger().setLevel(logging.INFO)
```

Comment:
a lot of `logging.getLogger()`. Looks not too much pythonic
I would consider (just consider ;)):
```
log = logging.getLogger()
log.addHandler(LogSenseHandler(logsense_token))
log.setLevel(logging.INFO)

...

# log.info(...)
```